### DISCUSSION TITLE
I there a real doc ?

### DISCUSSION THREAD

**Discussion by lepresk** [bdlukaa/fluent_ui#841] [Q&A] [answered]
Hello! I noticed that the link https://bdlukaa.github.io/fluent_ui/ only showcases the components of Fluent UI. Is there a separate documentation available that provides detailed information and usage guidelines for Fluent UI? I would appreciate any assistance or direction regarding the official documentation for Fluent UI. Thank you!

**Comment by bdlukaa** [comment]
Yes! Here's the one provided microsoft with guidelines and on: https://learn.microsoft.com/en-us/windows/apps/design/

**Comment by lepresk** [comment]
I'm talking about a place where there is a kind of Api reference of all the widgets, for example, I had to spend a lot of time searching here and there to realize that there are widgets like `ScaffoldPage` or `PageHeader` that can't be mentioned in the microsoft documentation

**Reply by bdlukaa** [reply, marked as answer]
hmm. the best you may get is the API reference (https://pub.dev/documentation/fluent_ui/latest/) and the example app (https://github.com/bdlukaa/fluent_ui/tree/master/example)

**Comment by WinXaito** [comment]
About that, I agree that we need a real documentation (not only a showcase or the autogenerated API).

But that's take much time, and need to be maintained.
